etc/encoding/opencast.properties

```diff
@@ -0,0 +1,243 @@
+#
+# profile.<format>.input:
+#     Track types for which this format is applicable.
+#     [audio|visual|audiovisual|stream|enhancedaudio|image|imagesequence|cover|nothing]
+#
+# profile.<format>.jobload:
+#     The job load for this specific encoding profile.
+##
+
+# The video distribution profile, optimized for parallel encoding of multiple
+# qualities for adaptive streaming
+profile.adaptive-parallel.http.name = parallel encode 480p and 720p and conditionally 1080p and 2160p qualities for adaptive streaming
+profile.adaptive-parallel.http.input = visual
+profile.adaptive-parallel.http.output = visual
+profile.adaptive-parallel.http.jobload = 4
+profile.adaptive-parallel.http.suffix.480p-quality = -480p.mp4
+profile.adaptive-parallel.http.suffix.720p-quality = -720p.mp4
+profile.adaptive-parallel.http.suffix.1080p-quality = -1080p.mp4
+profile.adaptive-parallel.http.suffix.1440p-quality = -1440p.mp4
+profile.adaptive-parallel.http.suffix.2160p-quality = -2160p.mp4
+profile.adaptive-parallel.http.ffmpeg.command.if-height-geq-900 = \
+  -filter:v scale=w=1920:h='trunc(1920/dar/2)*2',setsar=1,fps=25 -pix_fmt yuv420p \
+    -c:v libx264 -profile:v high -level 4.0 -crf 23 \
+    -x264opts keyint=25:min-keyint=25:no-scenecut -maxrate 4000k -bufsize 8000k \
+    -c:a aac -b:a 96k -ac 2 \
+    -movflags faststart #{out.dir}/#{out.name}#{out.suffix.1080p-quality}
+profile.studio.adaptive-parallel.http.ffmpeg.command.if-height-geq-1260 = \
+  -filter:v scale=w=2560:h='trunc(2560/dar/2)*2',setsar=1,fps=25 -pix_fmt yuv420p \
+    -c:v libx264 -profile:v high -level 5.0 -crf 23 \
+    -x264opts keyint=25:min-keyint=25:no-scenecut -maxrate 9800k -bufsize 19600k \
+    -c:a aac -b:a 96k -ac 2 \
+    -movflags faststart #{out.dir}/#{out.name}#{out.suffix.1440p-quality}
+profile.adaptive-parallel.http.ffmpeg.command.if-height-geq-1800 = \
+  -filter:v scale=w=3840:h='trunc(3840/dar/2)*2',setsar=1,fps=25 -pix_fmt yuv420p \
+    -c:v libx264 -profile:v high -level 5.1 -crf 23 \
+    -x264opts keyint=25:min-keyint=25:no-scenecut -maxrate 14800k -bufsize 29600k \
+    -c:a aac -b:a 96k -ac 2 \
+    -movflags faststart #{out.dir}/#{out.name}#{out.suffix.2160p-quality}
+profile.adaptive-parallel.http.ffmpeg.command = -i #{in.video.path} \
+  -filter:v scale=w=854:h='trunc(854/dar/2)*2',setsar=1,fps=25 -pix_fmt yuv420p \
+    -c:v libx264 -profile:v high -level 4.0  -crf 23 \
+    -x264opts keyint=25:min-keyint=25:no-scenecut -maxrate 800k -bufsize 1600k \
+    -c:a aac -b:a 64k -ac 2 \
+    -movflags faststart #{out.dir}/#{out.name}#{out.suffix.480p-quality} \
+  -filter:v scale=w=1280:h='trunc(1280/dar/2)*2',setsar=1,fps=25 -pix_fmt yuv420p \
+    -c:v libx264 -profile:v high -level 4.0  -crf 23 \
+    -x264opts keyint=25:min-keyint=25:no-scenecut -maxrate 1200k -bufsize 2400k \
+    -c:a aac -b:a 96k -ac 2 \
+    -movflags faststart #{out.dir}/#{out.name}#{out.suffix.720p-quality} \
+  #{if-height-geq-900} \
```

Comment:
So in this particular example, instead of 

`#{if-height-geq-900}`

it would be more desirable to use

`#{if-width-or-height-geq-1600-900}`

to avoid upscaling videos with sufficient height but absurdly small width?